experiments/imit_benchmark.sh

```diff
@@ -30,11 +31,11 @@ while true; do
       shift
       ;;
     --gail)
-      extra_configs+="gail "
+      ALGORITHM="gail"
```

Comment:
Feels a little odd this is a no-op since `ALGORITHM` is already set to gail, but it probably is better UI to keep the flag.

Reply:
Another option would be to force the user to choose GAIL or AIRL flags. Keeping as is for now.